module/buff/basic_buff.cpp

```diff
@@ -274,54 +270,55 @@ void Detector::bgrProcessing(const int& _my_color) {
     case uart::RED:
       fmt::print("[{}] Image pre-processing color: RED\n", idntifier_green);
 
-      cv::subtract(split_img_[2], split_img_[0], bin_img_color_);
+      /* my_color为红色，则处理红色的情况 */
+      /* 灰度图与RGB同样做红色处理 */
+      cv::subtract(split_img_[2], split_img_[0], bin_img_color_);  // r-b
 
+#ifndef RELEASE
       if (buff_config_.ctrl.IS_PARAM_ADJUSTMENT == 1) {
         std::string window_name = {"[basic_buff] bgrProcessing() -> trackbar"};
 
         cv::namedWindow(window_name);
-        cv::createTrackbar("GRAY_TH_RED:", window_name, &buff_config_.param.RED_BUFF_GRAY_TH, 255, nullptr);
-        cv::createTrackbar("COLOR_TH_RED:", window_name, &buff_config_.param.RED_BUFF_COLOR_TH, 255, nullptr);
+        cv::createTrackbar("GRAY_TH_RED:", window_name, &buff_config_.param.RED_BUFF_GRAY_TH, 255,
+                           nullptr);
+        cv::createTrackbar("COLOR_TH_RED:", window_name, &buff_config_.param.RED_BUFF_COLOR_TH, 255,
+                           nullptr);
 
         cv::imshow(window_name, trackbar_img_);
+        fmt::print("🧐 BGR红色预处理调参面板已打开 🧐\n", idntifier_green);
       }
+#endif  // !RELEASE
 
-      cv::threshold(gray_img_,
-                    bin_img_gray_,
-                    buff_config_.param.RED_BUFF_GRAY_TH,
-                    255,
+      cv::threshold(gray_img_, bin_img_gray_, buff_config_.param.RED_BUFF_GRAY_TH, 255,
                     cv::THRESH_BINARY);
-      cv::threshold(bin_img_color_,
-                    bin_img_color_,
-                    buff_config_.param.RED_BUFF_COLOR_TH,
-                    255,
+      cv::threshold(bin_img_color_, bin_img_color_, buff_config_.param.RED_BUFF_COLOR_TH, 255,
                     cv::THRESH_BINARY);
 
       break;
     case uart::BLUE:
       fmt::print("[{}] Image pre-processing color: BLUE\n", idntifier_green);
 
+      /* my_color为蓝色，则处理蓝色的情况 */
+      /* 灰度图与RGB同样做蓝色处理 */
       cv::subtract(split_img_[0], split_img_[2], bin_img_color_);  // b-r
```

Comment:
```suggestion
      // my_color 为蓝色，则处理蓝色的情况，灰度图与 RGB 同样做蓝色处理
      cv::subtract(split_img_[0], split_img_[2], bin_img_color_);  // b-r
```
中文与英文之间留空格，能单行说明白的避免写两行